content/en/docs/ambient/usage/waypoint/index.md

```diff
@@ -31,13 +31,26 @@ When you configure redirection to a waypoint, traffic will be forwarded by ztunn
 
 ## Deploy a waypoint proxy
 
-Waypoint proxies are deployed declaratively using Kubernetes Gateway resources. You can use istioctl experimental subcommands to generate, apply or list these resources.
+Waypoint proxies are deployed using standard Kubernetes Gateway resources. 
+
+You need to install the [Kubernetes Gateway API](https://gateway-api.sigs.k8s.io/) CRDs, which don’t come installed by default on most Kubernetes clusters:
+
+{{< text syntax=bash snip_id=install_k8s_gateway_api >}}
+$ kubectl get crd gateways.gateway.networking.k8s.io &> /dev/null || \
+  { kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.1.0/standard-install.yaml; }
+{{< /text >}}
+
```

Comment:
```suggestion
{{< boilerplate gateway-api-install-crds >}}
```

Reply:
The boilerplate uses a different snippet, which uses `experimental` and has a warning scare - which is why I didn't use it here.

What I suggest is that in a separate PR we rename `boilerplate gateway-api-install-crds` to `boilerplate gateway-api-install-experimental-crds` for the other usages of this boilerplate (which are all non-ambient I think).

Reply:
(oh we have several my bad yeah - I updated the "normal" one to skip `kustomize` which isn't used for the standard ones AFAICT and does nothing)